pkg/terraform/v1beta1/config.go

```diff
@@ -495,6 +496,11 @@ func setWorkersetFlag(w *kubeonev1beta1.DynamicWorkerConfig, name string, value
 		if s == nil {
 			return nil
 		}
+	case machinecontroller.AzureImagePlan:
```

Comment:
Probably not needed because we use the pointer type, but I added it anyway because I can't test is it going to work without it.